llvm/test/CodeGen/WebAssembly/exception-legacy.mir

```diff
@@ -78,7 +78,8 @@ body: |
     EH_LABEL <mcsymbol .Ltmp2>
     CATCHRET %bb.2, %bb.1, implicit-def dead $arguments
 
-  ; CHECK: bb.2
+  ; This BB should remain (it will be renumbered to bb.1)
+  ; CHECK: bb.1
```

Comment:
This renumbering happened because now we run `recalculateScopeTops` every time when EH is used. We used to run it only when unwind mismatches existed. Now we run it every time because adding `unreachable` after `end_try_table` requires splitting BBs and CFG changes.